chatbees_script.js

```diff
@@ -94,10 +94,15 @@ function chatbeesSendMessage() {
     },
     body: jsonData,
   })
-  .then(response => response.json())
+  .then((response) => {
+      if (response.ok) {
+         return response.json();
+      }
+      return Promise.reject(response);
```

Comment:
can we return the error in the response?

Reply:
yup, added.